client/my-sites/plugins/plugins-category-results-page/index.jsx

```diff
@@ -12,36 +13,35 @@ const PluginsCategoryResultsPage = ( { category, siteSlug, sites } ) => {
 
 	const categories = useCategories();
 	const categoryName = categories[ category ]?.name || category;
+	const categoryDescription = categories[ category ]?.categoryDescription;
 	const translate = useTranslate();
 
 	let title = '';
 	if ( categoryName && pagination ) {
-		title = translate(
-			'Found %(total)s plugin under "%(categoryName)s"',
-			'Found %(total)s plugins under "%(categoryName)s"',
-			{
-				count: pagination.results,
-				textOnly: true,
-				args: {
-					total: pagination.results,
-					categoryName,
-				},
-			}
-		);
+		title = translate( '%(total)s plugin', '%(total)s plugins', {
+			count: pagination.results,
+			textOnly: true,
+			args: {
+				total: pagination.results,
+			},
+		} );
 	}
 
 	return (
-		<FullListView
-			plugins={ plugins }
-			listName={ category }
-			title={ title }
-			site={ siteSlug }
-			showPlaceholders={ isFetching }
-			sites={ sites }
-			variant={ PluginsBrowserListVariant.InfiniteScroll }
-			fetchNextPage={ fetchNextPage }
-			extended
-		/>
+		<>
+			<Header title={ categoryName } count={ title } subtitle={ categoryDescription } />
+
+			<FullListView
+				plugins={ plugins }
+				listName={ category }
+				site={ siteSlug }
+				showPlaceholders={ isFetching }
+				currentSites={ sites }
```

Comment:
String freeze is going to take a while, could we ship this fix separately, sooner? (Thanks for fixing it btw)

Reply:
Actually, it should be `sites` instead of `currentSites`, I missed this while rebasing and resolving conflicts.